src/components/SubscribeNewsletter/subscribe-newsletter.tsx

```diff
@@ -0,0 +1,40 @@
+import { SquiggleLeft, SquiggleRight } from '@/assets/svg'
+import Button from '../Button/button'
+
+function SubscribeNewsLetter() {
+  return (
+    <div className="w-full relative bg-ibtc-light-yellow flex flex-col items-center justify-center min-h-[26rem] gap-[2rem]">
+      <div className="text-center space-y-[1rem]">
+        <h2 className="text-[2rem] lg:text-[3.438rem] font-[500] m-0 p-0">
```

Comment:
Use the class "_header" its content is ` ._header {
    @apply font-medium xl:text-[3.4rem]/[3.4rem] text-[2.4rem]/[2.4rem];
  }` instead.

Reply:
Noted. I'll update the PR soon.